src/components/Asset/AssetActions/Trade/FormTrade.tsx

```diff
@@ -185,28 +185,14 @@ export default function FormTrade({
     >
       {({ isSubmitting, setSubmitting, submitForm, values, isValid }) => (
         <>
-          {isWarningAccepted ? (
```

Comment:
same here, we do not need the `isWarningAccepted` state variable anymore

Reply:
same with trade